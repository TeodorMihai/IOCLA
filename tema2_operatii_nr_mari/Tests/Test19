00000001 * 1111111111
