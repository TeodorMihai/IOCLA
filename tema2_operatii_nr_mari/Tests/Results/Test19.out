01000000
1111111111
1111111111
1111111111
